core/sdk/src/api/manifest/device_manifest.rs

```diff
@@ -147,8 +147,14 @@ pub struct DistributionConfiguration {
 
 impl Default for DistributionConfiguration {
     fn default() -> Self {
+        let library_path = if cfg!(feature = "local_dev") {
```

Comment:
The default should be used if no env var overrides are set, so I think this code should remain as-is.